### DISCUSSION TITLE
Display preview is very weird sometimes

### DISCUSSION THREAD

**Discussion by Ethanout** [misode/misode.github.io#591] [Q&A]
![image](https://github.com/user-attachments/assets/6dc816b9-70be-47a3-9133-5de9b3fb858d)
![image](https://github.com/user-attachments/assets/2700e4cb-3836-410d-b468-35406d5b802a)
The preview is different from what's in the game sometimes

**Comment by Ethanout** [comment]
Shall I post it as an issue?

**Reply by misode** [reply]
Yes please, sorry that I haven't been able to look into this!
